deps/rabbit/test/config_schema_SUITE_data/rabbit.snippets

```diff
@@ -100,6 +100,38 @@ ssl_options.fail_if_no_peer_cert = true",
  {socket_writer_gc_threshold_off,
   "socket_writer.gc_threshold = off", [{rabbit, [{writer_gc_threshold, undefined}]}],[]},
 
+ {default_limits_vhost,
+ "
+  default_limits.vhost.a.pattern = .*
+  default_limits.vhost.a.max_queues = 10
+  default_limits.vhost.a.max_connections = 100
+ ",
+  [{rabbit, [{default_limits, [{vhost, [
+      {{re_pattern,0,0,0,
+                     <<69,82,67,80,73,0,0,0,0,0,0,0,1,129,0,0,255,255,255,255,
+                       255,255,255,255,0,0,0,0,0,0,0,0,0,0,64,0,0,0,0,0,0,0,0,
+                       0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,131,0,5,94,
+                       12,120,0,5,0>>},
+                    [{<<"max-connections">>,100},
+                     {<<"max-queues">>,10}]}
+                                       ]}]}]}],
+  []},
+
+ {default_limits_vhost_no_undefined,
+ "
+  default_limits.vhost.a.pattern = .*
+  default_limits.vhost.a.max_queues = 10
+ ",
+  [{rabbit, [{default_limits, [{vhost, [
+      {{re_pattern,0,0,0,
```

Comment:
Can we avoid matching against a record or at least compile it in the test using `re`? The record will change over time, it would be nice to not have to update this test every time it does.

Reply:
I see no easy way to inject code into s test snippet (rabbit_ct_config_schema Plugins seems to be a stub?). But upon reflection re:compile is unnecessary optimization in this context. I'll revert to using binaries.